release_tester/arangodb/starter/manager.py

```diff
@@ -57,7 +58,7 @@ def __init__(self,
             self.moreopts += ["--starter.port", "%d" % self.starter_port]
 
         self.hotbackup = []
-        if self.cfg.enterprise:
+        if self.cfg.enterprise and semver.compare(self.cfg.version, "3.5.1") >=0:
```

Comment:
this should rather be done at a central place:
```
release_tester/arangodb/starter/deployments/runner.py:        self.hot_backup = cfg.enterprise and self.supports_backup_impl() and self.old_installer.supports_hot_backup()
```
on the cfg object, and then used here.